boards/riscv/adp_xc7k_ae350/adp_xc7k_ae350_defconfig

```diff
@@ -13,6 +13,9 @@ CONFIG_SERIAL=y
 # GPIO driver options
 CONFIG_GPIO=y
 
+# I2C driver options
+CONFIG_I2C=y
+
```

Comment:
Don't y-select `CONFIG_I2C` by default. See this PR: https://github.com/zephyrproject-rtos/zephyr/pull/48574

Reply:
Thanks for reminding. Removed it